concrete/src/Validator/ValidatorServiceProvider.php

```diff
@@ -1,18 +1,20 @@
 <?php
+
 namespace Concrete\Core\Validator;
 
 use Concrete\Core\Foundation\Service\Provider;
 
 class ValidatorServiceProvider extends Provider
 {
     /**
-     * Registers the services provided by this provider.
+     * {@inheritdoc}
+     *
+     * @see \Concrete\Core\Foundation\Service\Provider::register()
```

Comment:
What's the point of the `@see` nodes in these docs? `{@inheritdoc}` is pulling from where you're `@see`ing, right?

Reply:
It's my IDE that adds this. And I think it's good: a class may inherit phpdoc from multiple interfaces and parent classes: with this `@see` phpdoc it's immediately clear where the method comes from, even for human eyes.

Reply:
I guess it's worth erring on the side of caution.